rollup/internal/controller/relayer/l1_relayer.go

```diff
@@ -150,6 +154,19 @@ func (r *Layer1Relayer) ProcessGasPriceOracle() {
 		}
 
 		if r.shouldUpdateGasOracle(baseFee, blobBaseFee, isCurie) {
+			// It indicates the commit/finalize batch has been stuck for a long time, it's likely that the L1 gas fee spiked.
```

Comment:
```suggestion
			// It indicates the committing batch has been stuck for a long time, it's likely that the L1 gas fee spiked.
```

Reply:
One risk: if committing is stuck for other reasons (e.g. bugs in the rollup-relayer), then the user will be charged by "default base fee and blob base fee" during halting (for newly generated batches).

Maybe setting the default value as Ethereum's average in the deployment's config.json could prevent fund loss in the above case.

Reply:
or we can query L1 gas price to see if it has a very high base fee or blob base fee.